[Websocket] Url without port is not supported

I use OpenShift. URLs in routes do not have ports. They look like `https://domain-a.corp.org`. So in java URL object port is -1. But netty does not support this .
```
Caused by: java.lang.IllegalArgumentException: port out of range:-1
        at java.net.InetSocketAddress.checkPort(InetSocketAddress.java:143)
        at java.net.InetSocketAddress.createUnresolved(InetSocketAddress.java:254)
        at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:123)
        at io.undertow.websockets.jsr.WebsocketConnectionBuilder.connect(WebsocketConnectionBuilder.java:202)
        at io.undertow.websockets.jsr.ServerWebSocketContainer.connectToServerInternal(ServerWebSocketContainer.java:543)
        at io.undertow.websockets.jsr.ServerWebSocketContainer.connectToServerInternal(ServerWebSocketContainer.java:520)
        at io.undertow.websockets.jsr.ServerWebSocketContainer.connectToServer(ServerWebSocketContainer.java:279)
        at mapackage.TestWebSocketService.connect(TestWebSocketService.java:32)
```

I found some mapping in undertow(original):

https://github.com/undertow-io/undertow/blob/master/core/src/main/java/io/undertow/websockets/client/WebSocketClient.java#L218


But it seems in quarkus there is no similar code

--- a/websockets-jsr/src/main/java/io/undertow/websockets/jsr/WebsocketConnectionBuilder.java
+++ b/websockets-jsr/src/main/java/io/undertow/websockets/jsr/WebsocketConnectionBuilder.java
@@ -187,7 +187,7 @@ class WebsocketConnectionBuilder {
                     public void initChannel(SocketChannel ch) throws Exception {
                         ChannelPipeline pipeline = ch.pipeline();
                         if (ssl != null) {
-                            SSLEngine sslEngine = ssl.createSSLEngine(uri.getHost(), uri.getPort());
+                            SSLEngine sslEngine = ssl.createSSLEngine(uri.getHost(), uri.getPort() == -1 ? (uri.getScheme().equals("wss") ? 443 : 80) : uri.getPort());
                             sslEngine.setUseClientMode(true);
                             pipeline.addLast("ssl", new SslHandler(sslEngine));
                         }

